src/Shortcodes/FileShortcodeProvider.php

```diff
@@ -136,17 +136,30 @@ public static function handle_shortcode($arguments, $content, $parser, $shortcod
      */
     protected static function getCachedMarkup($cache, $cacheKey, $arguments): string
     {
+        // Retrieve the cache markup
         $item = $cache->get($cacheKey);
+        if (empty($item) || empty($item['markup'])) {
```

Comment:
If `$item` is not empty, but `$item['markup']` is empty, surely this is bad cache rather than a miss? Since it means **something** was cached with this key, but not something with markup?

Reply:
Moved the markup check to be with the $filename and $hash check.